tests/bluetooth/bsim_bt/bsim_test_eatt/tests_scripts/reconfigure.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+simulation_id="reconfigure"
+verbosity_level=2
+process_ids=""; exit_code=0
+
+function Execute(){
+  if [ ! -f $1 ]; then
+    echo -e "  \e[91m`pwd`/`basename $1` cannot be found (did you forget to\
+ compile it?)\e[39m"
+    exit 1
+  fi
+  timeout 120 $@ & process_ids="$process_ids $!"
+}
+
+: "${BSIM_OUT_PATH:?BSIM_OUT_PATH must be defined}"
+
+#Give a default value to BOARD if it does not have one yet:
+BOARD="${BOARD:-nrf52_bsim}"
+
+cd ${BSIM_OUT_PATH}/bin
+
+Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_eatt_prj_autoconnect_conf \
+  -v=${verbosity_level} -s=${simulation_id} -d=0 -testid=central_reconfigure
+
+Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_eatt_prj_autoconnect_conf \
+  -v=${verbosity_level} -s=${simulation_id} -d=1 -testid=peripheral_reconfigure
+
+Execute ./bs_2G4_phy_v1 -v=${verbosity_level} -s=${simulation_id} \
+  -D=2 -sim_length=200e6 $@
```

Comment:
Doesn't matter much, but isn't `200e6` quite a long length to set for such a simple test? Or is this to handle large numbers of EATT bearers?

Reply:
Yes, that could be reduced. Locally the test finishes in under a second of in-simulation time. I think this was copy-pasted from another test that required longer time.

Reply:
Of course if you attempt to create 100 EATT bearers, it'll take longer :D